modules/vpc-sc/README.md

```diff
@@ -122,7 +130,7 @@ module "test" {
     r1 = {
       spec = null
       status = {
-        access_levels       = [module.test.access_level_names["a1"]]
+        access_levels       = [module.test.access_level_names["a1"], "a2"]
```

Comment:
access_levels = [module.test.access_level_names["a1"], module.test.access_level_names["a2"]] ?

Reply:
aaaah right!

Reply:
lol, the point of this PR is exactly to do what is shown above...